autoload/neomake/makers/ft/erlang/gradualizer.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+#set -x
+
+# Set to your gradualizer location or make sure the command is in PATH
+GRADUALIZER=gradualizer
```

Comment:
Maybe this could be `gradualizer=${GRADUALIZER:-gradulizer}` (and then use `$gradulizer` below), if this should be more easily configurable.

Also consider using `NEOMAKE_GRADUALIZER` first then, if it makes sense.